config/minor_filter.txt

```diff
@@ -41,4 +41,5 @@ hbu=how about you
 \bur\b=your
 btw=by the way
 pls=please
-plz=please
\ No newline at end of file
+plz=please
+tf=the fuck
```

Comment:
```suggestion
\btf\=the fuck
```

Reply:
i assume this makes it require it not being part of a word, since there's probably a word out there that uses "tf"

Reply:
its the inverse actually

Reply:
It isnt, \b \b makes it require spaces